ui/apps/platform/cypress/integration/collections/Collections.helpers.js

```diff
@@ -0,0 +1,81 @@
+import navSelectors from '../../selectors/navigation';
+
+import { visitFromLeftNavExpandable } from '../../helpers/nav';
+import { visit } from '../../helpers/visit';
+import { collectionSelectors } from './Collections.selectors';
+
+const basePath = '/main/collections';
+
+export const collectionsAlias = 'collections';
+export const collectionsCountAlias = 'collections/count';
+
+const routeMatcherMapForCollections = {
+    [collectionsAlias]: {
+        method: 'GET',
+        url: '/v1/collections?query.query=*',
+    },
+    [collectionsCountAlias]: {
+        method: 'GET',
+        url: '/v1/collectionscount?query.query=*',
+    },
+};
+
+// visit
+
+export function visitCollections(staticResponseMap) {
+    visit(basePath, routeMatcherMapForCollections, staticResponseMap);
+
+    cy.get('h1:contains("Collections")');
+    cy.get(`${navSelectors.navExpandable}:contains("Platform Configuration")`);
+    cy.get(`${navSelectors.nestedNavLinks}:contains("Collections")`).should(
```

Comment:
As a follow up, we (probably me) might write an assertion helper function:
* Encapsulate navigation selectors so simple `selectors` can imply collection selectors.
* Increase parallel form with `visitFromLeftNavExpandable` below.

What do you think about:

```js
assertLeftNavExpandable('Platform Configuration', 'Collections');
```

Reply:
Sure, or maybe `assertLeftNavCurrentPage()`?